README.rst

```diff
@@ -91,33 +91,18 @@ buffering a point, using the scalar Geometry interface:
     >>> patch.area
     313.6548490545941
 
-Using the vectorized ufunc interface, compare a grid of points with a polygon:
+Using the vectorized ufunc interface (instead of using a manual for loop),
+compare an array of points with a polygon:
 
 .. code:: python
 
-    >>> geoms = shapely.points(*np.indices((4, 4)))
+    >>> import shapely
+    >>> import numpy as np
+    >>> geoms = np.array([Point(0, 0), Point(1, 1), Point(2, 2)])
```

Comment:
I changed the array construction to a manual array instead of the `shapely.points` constructor, as I think this example should focus on showing the vectorized contains (and that way it is also more visual already what the array looks like)